### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by razinc**
+ Advantages of Using PowerShell

### CODE REVIEW

**Review by joeyaiello — CHANGES_REQUESTED** [review]
I'm actually surprised these "Learning PowerShell" docs are still here. Ideally, all of this content should be moved to PowerShell/PowerShell-Docs (possibly somewhere like [here](https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting?view=powershell-6)

In the meantime, I'd rather not continue to add to these (rather, we should subtract and push it over to docs.microsoft.com)

**Comment by iSazonov** [comment]
@SteveL-MSFT Who can address the @joeyaiello request? I think we should close the PR.

**Comment by iSazonov** [comment]
@joeyaiello Please update your review.

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ You won't be a PowerShell guru at the end of reading this material but you will
 
 If you have 30 minutes now, let’s try it.
 
+Advantages of Using PowerShell
```
Comment (TravisEz13) [inline docs/learning-powershell/README.md]: @razinc 	or @RDIL 	

Could you clean up this file with markdownlint?

The rules are in the root of the repo.  There is a Visual Studio code extension for markdown lint which works well if you open from the root of the repo.

**Comment by iSazonov** [comment]
I'd remove the file and add a reference on new docs to root readme.md.

**Comment by iSazonov** [comment]
@joeyaiello @SteveL-MSFT What is conclusion for the PR?

**Comment by RDIL** [comment]
@SteveL-MSFT @HemantMahawar @joeyaiello ?

**Comment by TravisEz13** [comment]
This doc should be migrated.  See https://github.com/PowerShell/PowerShell/pull/7952#issuecomment-457776829

**Pull request closed without merging**